guides/Makefile

```diff
@@ -13,10 +13,10 @@ pdf: all-pdf
 clean: all-clean
 
 linkchecker:
-	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
+	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern --verbose
 
 linkchecker-tryer:
-	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern | linkchecker-tryer
+	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern --verbose | linkchecker-tryer
 
```

Comment:
I am adding verbosity to be able to verify links are actually rendered as expected (locally).